src/JestProcessManagement/JestProcess.ts

```diff
@@ -68,9 +68,12 @@ export class JestProcess {
     this.stopResolveCallback = null
     let exited = false
 
+    const reporterPath = `${process.cwd()}/src/JestProcessManagement/vscode-jest-reporter.ts`
     const options = {
       noColor: true,
       shell: platform() === 'win32',
+      // workaround required until reporters argument won't be supported by jest-editor-support
+      testFileNamePattern: `--reporters="${reporterPath}"`,
```

Comment:
can jest actually run a typescript reporter? I suspect you will have to pass the js file instead...

also, I know this is just a temporary workaround, once the implementation worked, we should use a new option argument and patch `jest-editor-support` accordingly.

Reply:
I'll try with that TS in .js idea. Pure .ts file didn't work. As for workaround I'll make PR to jest-editor-support when this PR will be ready.

Reply:
It seems to be working with `.ts` reporter file.